JavaScript/JavaScriptSDK.Tests/CheckinTests/Telemetry/RemoteDependency.tests.ts

```diff
@@ -38,6 +38,22 @@ class RemoteDependencyTests extends ContractTestHelper {
         });
 
         this.testCase({
+            name: name + "Command name is truncated if too long",
+            test: () => {
+                var urlLength = 2049;
+                var longUrl = "";
+                for (var i = 0; i < urlLength; i++) {
+                    longUrl += "A";
```

Comment:
Just for fun I wanted to share these alternatives:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
http://stackoverflow.com/questions/1877475/repeat-character-n-times